biom/tests/test_cli/test_subset_table.py

```diff
@@ -55,9 +55,10 @@ def test_invalid_input(self):
     def test_subset_samples_hdf5(self):
         """Correctly subsets samples in a hdf5 table"""
         cwd = os.getcwd()
-        if '/' in __file__:
-            os.chdir(__file__.rsplit('/', 1)[0])
-        obs = _subset_table(hdf5_biom='test_data/test.biom', axis='sample',
+        if os.path.sep in __file__:
+            os.chdir(__file__.rsplit(os.path.sep, 1)[0])
```

Comment:
`__file__.rsplit(os.path.sep, 1)[0]` can be `os.path.dirname(__file__)`, I guess.